core-bundle/src/Resources/contao/languages/en/tl_page.xlf

```diff
@@ -431,6 +437,9 @@
       <trans-unit id="tl_page.map_never">
         <source>Show never</source>
       </trans-unit>
+      <trans-unit id="tl_page.layout_inherit">
```

Comment:
This should probably be camelCase.

Reply:
I went with snake case to be in line with labels like `map_never` (the line above), somewhat indicating the `inherit` part is an option of `layout`.

But either is fine with me.